c_src/js.cc

```diff
@@ -16,6 +16,9 @@
 
 #include "errors.h"
 
+#define ACTIVE_SLEEP_TIME 100000 // 100,000 us = 100ms = 0.1s
+#define MAX_SLEEP_TIME INT_MAX // A really long time
```

Comment:
Minor nit: if `INT_MAX` is `2**31` it will be about 30min. Probably not an issue, unless we need it to sleep effectively "forever".

Reply:
Its just for CPU savings so that an idle ateles isn't doing all that much. One could argue that 10 calls a second isn't all that big of a deal but I copied the general pattern from the `shell/js.cpp` example in Spidermonkey.  However, I've also made it much longer now due to your other question.